tools/unwrapfile/unwrapfile.cc

```diff
@@ -181,76 +176,111 @@ void unwrapMP4Frames(const char *filepathRaw, unsigned int frameInterval, const
 }
 }   // Anonymous namespace
 
+void usage()
+{
+    std::cout << "usage: unwrapfile [--no-sound] [--extract-frames[=SKIP]] [--camera CAMERA_NAME] [--unwrapped-resolution WIDTHxHEIGHT] [file(s)]" << std::endl;
+}
+
 int main(int argc, char** argv)
 {
     if (argc == 1) {
-        std::cout << "usage: unwrapfile [--no-sound] [--extract-frames SKIP] [--camera CAMERA_NAME] [--unwrapped-resolution WIDTH HEIGHT] [file(s)]" << std::endl;
-        return 0;
+        usage();
+        return EXIT_FAILURE;
     }
 
-    std::vector<FileType> ftype(argc - 1, FileType::skip);
-    bool anyvideo = false;
+    enum Args
+    {
+        NO_SOUND,
+        EXTRACT_FRAMES,
+        CAMERA,
+        UNWRAPPED_RESOLUTION,
+        HELP
+    };
+
+    CSimpleOpt::SOption clOpts[] = {
+        { NO_SOUND, "--no-sound", SO_NONE },
+        { EXTRACT_FRAMES, "--extract-frames", SO_OPT },
+        { CAMERA, "--camera", SO_NONE },
+        { UNWRAPPED_RESOLUTION, "--unwrapped-resolution", SO_REQ_SEP },
+        { HELP, "-h", SO_NONE },
+        { HELP, "--help", SO_NONE },
+        SO_END_OF_OPTIONS
+    };
+
     bool copysound = true;
-    bool extractFrames = true;
+    bool extractFrames = false;
```

Comment:
@tenxlenx and I encountered this bug as well - apologies for, presumably, fixing it on my machine and not committing...